kubernetes/annotations_and_labels.md

```diff
@@ -48,6 +49,7 @@ This page defines common annotations and labels we set in Kubernetes objects.
   statusresource.
 - `giantswarm.io/provider` - value should be the installation's provider, e.g.
   `kvm`, `aws`, or `azure`.
+- `version` - value should contain the version of the application.
 
```

Comment:
In sections below we should specify where those new labels are used. `version` is used wherever `app` is used and `giantswarm.io/reconciled-version` in operator App CRs, Deployments & Pods to begin with I think.

Reply:
`app` and `version` should be applied to every k8s resources which is an app. I'm not sure how to phrase this not to confuse with managed app, any idea? Or we just state as in `app` that is applies to every k8s resource?

Reply:
Good point. I made a suggestion here https://github.com/giantswarm/fmt/pull/81/files#r348469386.